src/migration/about.md

```diff
@@ -0,0 +1,5 @@
+# Migration
+
+The migration is a one-time transfer of existing bid data from Fleek to MongoDB. The switch to a more centralized database system is to increase scalability
+but is temporary as when an Ethereum L2 solution is available we will focus on adopting that to move back towards a more decentralized approach that allows for
+the scalability we're looking for.
```

Comment:
Has the script to perform the migration been created yet? This folder/file just seems to have code to fetch a file from Fleek, which isn't migrating anything.

Reply:
No no, not created yet. Actually this shouldn't have even committed 🤔